Source/Core/Core/HW/WiimoteEmu/Dynamics.h

```diff
@@ -45,6 +46,10 @@ struct IMUCursorState
 
   // Rotation of world around device.
   Common::Quaternion rotation;
```

Comment:
See my other comments as well.  If you're going to use orientation for the cursor, you probably want to remove this.  Or at least, if I'm understanding correctly, I wouldn't have both.